src/Pages/ContentListTable/ContentListTable.test.tsx

```diff
@@ -89,7 +89,8 @@ it('Render with a single row', () => {
       meta: { count: 1, limit: 20, offset: 0 },
     },
   }));
-
+});
+test('should trigger snapshot when kebab is clicked', async () => {
```

Comment:
remove these lines, the test is already named "Render with a single row" 
If you want to make the function async, simply add: 
`it('Render with a single row', **async** () => {`
on line 76.